pinot-core/src/main/java/com/linkedin/pinot/core/io/writer/impl/v1/FixedByteChunkSingleValueWriter.java

```diff
@@ -68,37 +68,44 @@ public FixedByteChunkSingleValueWriter(File file, ChunkCompressor compressor, in
 
   @Override
   public void setInt(int row, int value) {
```

Comment:
General question: argument row is ignored, so the behavior of this writer is different with old writers which implements SingleColumnSingleValueWriter interface. Do you plan to handle that?